src/loader/cmi.ml

```diff
@@ -332,12 +332,7 @@ let mark_class_declaration cld =
   List.iter mark_type_parameter cld.cty_params;
   mark_class_type cld.cty_params cld.cty_type
 
-let canonical doc : [`Dot of Path.Module.t * string] option =
-  let doc = List.map Odoc_model.Location_.value doc in
-  match List.find (function `Tag (`Canonical _) -> true | _ -> false) doc with
-  | exception Not_found -> None
-  | `Tag (`Canonical (`Dot (p, n))) -> Some (`Dot (p, n) )
-  | _ -> None
+(* Handle top-comments *)
```

Comment:
I don't think this comment should be here

Reply:
Indeed, removed.